website/src/segment.css.ts

```diff
@@ -63,7 +62,15 @@ export const wordGroup = style([
     "@media": {
       [mediaQueries.medium]: {
         display: "inline-block",
-        padding: 0,
+      }
+    }
+  },
+])
+
+export const wordGroupSelection = styleVariants({
```

Comment:
Not sure where in the styling this happens, but words get all messed up when you select them. [I'm looking here](https://deploy-preview-147--dailp.netlify.app/documents/dd6); when I select one word, then select another they shift around in the document. We really can't have that, we need the words to sit still when we (de)select them.

Looks like it might be this: You can't change the padding or margin between the selected and unselected states because that will change the layout flow.